### DISCUSSION TITLE
How to keep the server running?

### DISCUSSION THREAD

**Discussion by shorts7wound** [KiraLT/torrent-stream-server#254] [Q&A] [answered]
when i use the command it works but as soon as i leave my ssh it stops wrking
i am begginer in npm and need this for a project please help

**Comment by KiraLT** [comment, marked as answer]
To keep the server running you need to run this server with some kind of process/apps manager.

The recommended way is to use Kubernetes or plain Docker. But if you are not familiar with these technologies, you can just use NodeJS [pm2 package](https://www.npmjs.com/package/pm2). 

You can find more information in [setup docs](https://github.com/KiraLT/torrent-stream-server/wiki/Setup#vps)
